qiskit/circuit/library/standard_gates/ecr.py

```diff
@@ -21,11 +21,12 @@
 
 
 class ECRGate(Gate):
-    r"""An echoed RZX(pi/2) gate implemented using RZX(pi/4) and RZX(-pi/4).
+    r"""An echoed cross-resonance gate.
 
     This gate is maximally entangling and is equivalent to a CNOT up to
     single-qubit pre-rotations. The echoing procedure mitigates some
     unwanted terms (terms other than ZX) to cancel in an experiment.
+    More specifically, this gate implements :math:`\frac{1}{\sqrt{2}}{IX-XY}`.
```

Comment:
```suggestion
    More specifically, this gate implements :math:`\frac{1}{\sqrt{2}}(IX-XY)`.
```

Reply:
With all the confusion of qiskit ordering we might want to explicitly write it in tensor product notation `I\otimes X` etc. I'm not sure what convention we use in our other gate docs, but as I wrote it I was using `I \otimes X`, and `X \otimes Y` correspond to `target \otimes control` or`Q1 \otimes Q0` as the matrix def returned by `to_matrix`

Reply:
I personally don't particularly mind either way there - I think using whatever string order `Pauli` uses would be fine to match - so I'm happy to defer to you. I was just correcting the delimiters in the LaTeX expression.